app/views/catalog/_document.atom.builder

```diff
@@ -21,7 +21,7 @@ xml.entry do
   with_format(:html) do
     xml.summary "type" => "html" do
       document_component = blacklight_config.view_config(:atom).summary_component
-      xml.text! render document_component.new(presenter: document_presenter(document), component: :div, show: true)
+      xml.text! render document_component.with_individual(document, presenter: document_presenter(document), component: :div, show: true)
```

Comment:
Do we need to pass the document now?

Reply:
Pushed a commit removing `with_individual` and passing only the presenter here.